### PR TITLE
Add integration test to configure diagnostic severity via editorconfig

### PR DESCRIPTION
**Pull request by mavasani**

### CODE REVIEW

**Comment by mavasani** [comment]
Ping @jasonmalinowski @dotnet/roslyn-ide

File: src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs [hunk 1]
```diff
@@ -711,5 +712,63 @@ static void Main(string[] args)
...
+                    "Suppress CS0168",
+                        "in Source",
+                    "Configure CS0168 severity",
+                        "None",
+                        "Silent",
+                        "Suggestion",
+                        "Warning",
+                        "Error",
+            };
+            VisualStudio.Editor.Verify.CodeActions(expectedItems, applyFix: "Error", ensureExpectedItemsAreOrdered: true);
```
Comment (jasonmalinowski) [inline src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs:749]: Is there some way to have this validate the preview?

Comment (mavasani) [reply src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs:749]: I tried to thread in a delay between the action being computed and applied, and the integration test does wait after bringing up the light bulb, but the integration test does not expand the nested actions and hence do not bring up the preview. We either need to figure out some other way to test preview OR write a dummy test analyzer that registers a top level code action to update the .editorconfig, though I am not sure if our integration tests can run with a custom defined analyzer that is not in the box.

In short, I could not find an easy way to validate the preview.

**Comment by mavasani** [comment]
I'll merge this test as it seems useful by itself, even if it doesn't validate the preview currently - it will still validate applying a code fix from IDE that updates editorconfig and validates the change in reported diagnostic severity.
